specification/ml/evaluate_data_frame/MlEvaluateDataFrameRequest.ts

```diff
@@ -37,16 +37,15 @@ export interface Request extends RequestBase {
   body: {
     /**
      * Defines the type of evaluation you want to perform.
-     * @doc_url https://www.elastic.co/guide/en/elasticsearch/reference/current/evaluate-dfanalytics.html#ml-evaluate-dfanalytics-resources
+     * See [Data frame analytics evaluation resources](https://www.elastic.co/guide/en/elasticsearch/reference/current/evaluate-dfanalytics.html#ml-evaluate-dfanalytics-resources).
```

Comment:
This is just a link to a lower section of the API page, so IMO it's not useful here
```suggestion
```